src/gui/accountmanager.h

```diff
@@ -55,14 +56,15 @@ class AccountManager : public QObject {
 
     /**
      * Return a list of all accounts.
-     * (this is a list of QSharedPointer for internal reasons, one should normally not keep a copy of them)
+     * (this is a list of QSharedPointer for internal reasons, one should normally not keep a copy
+     * of them)
```

Comment:
@jturcotte Not setting the column limit is good - I agree that's not been an annoyance so far. Also reduces the number of affected lines from 9k to 7k.